src/main/java/it/gov/pagopa/payhub/activities/exception/custom/OperatorNotAuthorizedException.java

```diff
@@ -0,0 +1,11 @@
+package it.gov.pagopa.payhub.activities.exception.custom;
```

Comment:
every exception created is a custom...
we could partition by entities or use cases.
But this Exception is so generic that could live just under exception package